### PR TITLE
Dataset searching tests

### PR DESCRIPTION
**Pull request by jordanCain**

### CODE REVIEW

File: tests/FVTTests/constants.ts [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+ * https://www.eclipse.org/legal/epl-v20.html
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Copyright IBM Corporation 2020
+ */
+export const USERNAME :string = process.env.ZOWE_USERNAME;
+export const PASSWORD :string = process.env.ZOWE_PASSWORD;
+
+export const BASE_URL :string = `https://${process.env.SERVER_HOST_NAME}:${process.env.SERVER_HTTPS_PORT}`;
```
Comment (NakulManchanda) [inline tests/FVTTests/constants.ts:13]: can be used value from above USERNAME PASSWORD, instead of reaching out to process.env again

Comment (NakulManchanda) [reply tests/FVTTests/constants.ts:13]: instead of constants.ts can we call this file config.ts  or environment.ts

File: scripts/prepare-fvt.sh [hunk 2]
```diff
@@ -139,28 +139,12 @@ services:
     - https://${FVT_GATEWAY_HOST}:${FVT_API_PORT}/
   homePageRelativeUrl:  # Home page is at the same URL
   routedServices:
-    - gatewayUrl: api/v1  # [api/ui/ws]/v{majorVersion}
-      serviceRelativeUrl: api/v1/datasets
+    - gatewayUrl: api/v2  # [api/ui/ws]/v{majorVersion}
```
Comment (NakulManchanda) [inline scripts/prepare-fvt.sh:142]: gateway url api/v1 will change as well? 
my impression was only serviceRelativeUrl changes to api/v2

Comment (jordanCain) [reply scripts/prepare-fvt.sh:142]: This is for api mappings, nothing to do with the ui, essentially prior to this PR we never had tests for mvs explorer that checked any functionality, just that the page loaded so this didn't cause a problem up until now.

File: tests/FVTTests/constants.ts (hunk 1 above)
Comment (NakulManchanda) [reply tests/FVTTests/constants.ts:13]: sorry, i was reading wrong SERVER_HOST_NAME, SERVER_HTTPS_PORT as username password!

**Pull request merged**